README.md

```diff
@@ -71,8 +71,8 @@ helm delete --namespace test my-application
 | deployment.fluentdConfigAnnotations | object | `nil` | Configuration details for fluentdConfigurations. Only works for specific setup, see <https://medium.com/stakater/dynamic-log-processing-with-fluentd-konfigurator-and-slack-935a5de4eddb>. |
 | deployment.replicas | int | `nil` | Number of replicas. |
 | deployment.imagePullSecrets | list | `[]` | List of secrets to be used for pulling the images. |
-| deployment.envFrom | object | `nil` | Mount environment variables from ConfigMap or Secret to the pod. |
-| deployment.env | object | `nil` | Environment variables to be added to the pod. |
+| deployment.envFrom | object | `nil` | Mount environment variables from ConfigMap or Secret to the pod. See the README "Consuming environment variable in application chart" section for more details. |
+| deployment.env | object | `nil` | Environment variables to be added to the pod. See the README "Consuming environment variable in application chart" section for more details. |
```

Comment:
```suggestion
| deployment.envFrom | object | `nil` | Mount environment variables from ConfigMap or Secret to the pod. See the [Consuming environment variable in application chart](#consuming-environment-variable-in-application-chart) section for more details. |
| deployment.env | object | `nil` | Environment variables to be added to the pod. See the [Consuming environment variable in application chart](#consuming-environment-variable-in-application-chart) section for more details. |
```

WDYT @aslafy-z?

Reply:
Yeah it works too but I often reads the value file itself and it is a bit misleading in this case